internal/private/private.go

```diff
@@ -112,10 +112,10 @@ type TryReusingBlobOptions struct {
 	// Transports, OTOH, MUST support these fields being zero-valued for types.ImageDestination callers
 	// if they use internal/imagedestination/impl.Compat;
 	// in that case, they will all be consistently zero-valued.
-
-	EmptyLayer bool            // True if the blob is an "empty"/"throwaway" layer, and may not necessarily be physically represented.
-	LayerIndex *int            // If the blob is a layer, a zero-based index of the layer within the image; nil otherwise.
-	SrcRef     reference.Named // A reference to the source image that contains the input blob.
+	RequiredCompression *compression.Algorithm // If set the function will check if the blob which is being used as the candidate has the required compression.
```

Comment:
```suggestion
	RequiredCompression *compression.Algorithm // If set, only reuse blobs with the required compression.
```
or see elsewhere about the interaction with `CanSubstitute`.

Reply:
Done.